### PR TITLE
[Add Package] Lyrebird

### PR DESCRIPTION
**Pull request by D-Brox**
This PR adds `lyrebird`

### CODE REVIEW

File: packages/lyrebird/lyrebird.pacscript [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+pkgdir="/usr/src/pacstall/${pkgname}"
+removescript="yes"
+
+prepare() {
+    sudo pip3 -q install toml
+}
+
+build() {
+    BIN_PATH="${pkgdir}/usr/bin" SHARE_PATH="/usr/share/${pkgname}" envsubst < "${pkgname}.desktop" > auxfile
+    cat auxfile > "${pkgname}.desktop"
```
Comment (wizard-28) [inline packages/lyrebird/lyrebird.pacscript:18]:
```suggestion
    sudo BIN_PATH="${pkgdir}/usr/bin" SHARE_PATH="/usr/share/${pkgname}" envsubst < "${pkgname}.desktop" > auxfile
    sudo cat auxfile > "${pkgname}.desktop"
```

File: packages/lyrebird/lyrebird.pacscript [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+install() {
+    sudo install -Dm644 "${pkgname}.desktop" -t "${pkgdir}/usr/share/applications"
+    sudo install -Dm644 "icon.png" -t "${pkgdir}/usr/share/${pkgname}"
+    cp -rf "app/" "${pkgdir}/usr/share/${pkgname}"
```
Comment (wizard-28) [inline packages/lyrebird/lyrebird.pacscript:24]:
```suggestion
    sudo cp -rf "app/" "${pkgdir}/usr/share/${pkgname}"
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Does not seem to work?
Here's the log:
```
Module module-null-sink not loaded.
Module module-remap-source not loaded.
Traceback (most recent call last):
  File "/usr/src/pacstall/lyrebird/usr/share/lyrebird/app/mainwindow.py", line 256, in preset_clicked
    self.sox_process = subprocess.Popen(command.split(' '))
  File "/usr/lib/python3.8/subprocess.py", line 854, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.8/subprocess.py", line 1702, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'sox'
Module module-null-sink not loaded.
Module module-remap-source not loaded.
```

**Comment by D-Brox** [comment]
I need to check the dependencies, then

**Pull request closed without merging**